gef-ui/server/utils/MOCKS/index.js

```diff
@@ -0,0 +1,731 @@
+    securityDetails: {},
+    status: CONSTANTS.DEAL_STATUS.IN_PROGRESS,
+    requiredFields: [],
+  },
+  ukefDealId: '0030113304',
+  checkerId: '619bae3467cc7c002069fc21',
+  editedBy: ['619bae3467cc7c002069fc1e'],
+  additionalRefName: null,
+  facilitiesUpdated: 1638542220497,
+  comments: [],
+  previousStatus: CONSTANTS.DEAL_STATUS.UKEF_IN_PROGRESS,
+  ukefDecision: [],
+  ukefDecisionAccepted: true,
+  id: '61a7710b2ae62b0013dae687',
+  exporter: { status: CONSTANTS.DEAL_STATUS.COMPLETED, details: [], validation: [] },
+  facilities: {
+    status: CONSTANTS.DEAL_STATUS.COMPLETED,
+    items: [
+      {
+        status: CONSTANTS.DEAL_STATUS.COMPLETED,
+        details: {
+          _id: '61a771cc2ae62b0013dae68a',
+          dealId: '61a7710b2ae62b0013dae687',
+          type: 'CASH',
+          hasBeenIssued: false,
+          name: 'Facility two',
+          shouldCoverStartOnSubmission: true,
+          coverStartDate: 1638403200000,
+          coverEndDate: '2030-01-01T00:00:00.000Z',
+          monthsOfCover: null,
+          details: [],
+          detailsOther: '',
+          currency: 'GBP',
+          value: 2000,
+          coverPercentage: 80,
+          interestPercentage: 1,
+          paymentType: 'IN_ADVANCE_MONTHLY',
+          createdAt: 1638363596947,
+          updatedAt: 1638442632540,
+          ukefExposure: 1600,
+          guaranteeFee: 0.9,
+          submittedAsIssuedDate: '1638363717231',
+          ukefFacilityId: '0030113305',
+          feeType: 'in advance',
+          feeFrequency: 'Monthly',
+          dayCountBasis: 365,
+          coverDateConfirmed: false,
+        },
+        validation: { required: [] },
+      },
+    ],
+  },
+  exporterStatus: { text: 'Completed', class: 'govuk-tag--green', code: 'COMPLETED' },
+  eligibilityCriteriaStatus: { text: 'Completed', class: 'govuk-tag--green', code: 'COMPLETED' },
+  facilitiesStatus: { text: 'Completed', class: 'govuk-tag--green', code: 'COMPLETED' },
+  supportingInfoStatus: { text: 'Completed', class: 'govuk-tag--green', code: 'COMPLETED' },
+  canSubmit: false,
+  checkerCanSubmit: false,
+  maker: {
+    username: 'BANK1_MAKER1',
+    firstname: 'ABC',
+    surname: 'DEF',
+    email: 'test',
+    roles: [],
+    bank: {},
+    timezone: 'Europe/London',
+    lastLogin: '1638807320335',
+    'user-status': 'active',
+    _id: '619bae3467cc7c002069fc1e',
+  },
+  checker: {
+    username: 'BANK1_CHECKER1',
+    firstname: 'DEF',
+    surname: 'GHJ',
+    email: 'test2',
+    roles: ['maker'],
+    bank: {},
+    timezone: 'Europe/London',
+    lastLogin: '1638964634607',
+    'user-status': 'active',
+    _id: '619bae3467cc7c002069fc21',
+  },
+};
+
+module.exports = {
+  MOCK_AIN_APPLICATION,
+  MOCK_AIN_APPLICATION_UNISSUED_ONLY,
+  MOCK_UNISSUED_FACILITY,
+  MOCK_FACILITY,
+  MOCK_DEAL,
+  MOCK_REQUEST,
+  MOCK_ISSUED_FACILITY,
+  MOCK_AIN_APPLICATION_ISSUED_ONLY,
+  MOCK_ISSUED_FACILITY_UNCHANGED,
+  MOCK_AIN_APPLICATION_FALSE_COMMENTS,
+};
```

Comment:
We should split these out instead of adding all in one file - i.e have facilities in it's own file, same iwth `MOCK_REQUEST`. Not a blocker can be improved later

Reply:
part of dtfs2-5228 which I will do